src/Faker/Provider/bn_BD/PhoneNumber.php

```diff
@@ -7,7 +7,7 @@ class PhoneNumber extends \Faker\Provider\PhoneNumber
     public function phoneNumber()
     {
         $number = "+880";
-        $number .= static::randomNumber(7);
+        $number .= static::numerify($string = "##########");
```

Comment:
I don't think this does what you think it does. PHP doesn't have named parameters, so the `$string = ` is unnecessary.